diff --git a/node_modules/react-native-get-music-files/android/build.gradle b/node_modules/react-native-get-music-files/android/build.gradle
index bce0565..13e28cd 100644
--- a/node_modules/react-native-get-music-files/android/build.gradle
+++ b/node_modules/react-native-get-music-files/android/build.gradle
@@ -16,8 +16,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "29.0.2"
 
     defaultConfig {
         minSdkVersion 17
@@ -40,9 +40,8 @@ repositories {
 
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
-    compile files('src/main/libs/jid3lib-0.5.4.jar')
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
+    implementation files('src/main/libs/jid3lib-0.5.4.jar')
 }
-  
\ No newline at end of file
