diff --git a/node_modules/react-native-get-music-files/android/build.gradle b/node_modules/react-native-get-music-files/android/build.gradle
index bce0565..13e28cd 100644
--- a/node_modules/react-native-get-music-files/android/build.gradle
+++ b/node_modules/react-native-get-music-files/android/build.gradle
@@ -16,8 +16,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "29.0.2"
 
     defaultConfig {
         minSdkVersion 17
@@ -40,9 +40,8 @@ repositories {
 
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
-    compile files('src/main/libs/jid3lib-0.5.4.jar')
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
+    implementation files('src/main/libs/jid3lib-0.5.4.jar')
 }
-  
\ No newline at end of file
diff --git a/node_modules/react-native-get-music-files/android/src/main/java/com/cinder92/musicfiles/RNReactNativeGetMusicFilesModule.java b/node_modules/react-native-get-music-files/android/src/main/java/com/cinder92/musicfiles/RNReactNativeGetMusicFilesModule.java
index 4907263..10d84e7 100644
--- a/node_modules/react-native-get-music-files/android/src/main/java/com/cinder92/musicfiles/RNReactNativeGetMusicFilesModule.java
+++ b/node_modules/react-native-get-music-files/android/src/main/java/com/cinder92/musicfiles/RNReactNativeGetMusicFilesModule.java
@@ -17,9 +17,12 @@ import android.net.Uri;
 import android.os.Build;
 import android.os.Environment;
 import android.provider.MediaStore;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
+import android.util.Base64;
 import android.util.Log;
 
+import java.io.ByteArrayOutputStream;
+
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.WritableArray;
 import com.facebook.react.bridge.WritableMap;
@@ -147,7 +150,7 @@ public class RNReactNativeGetMusicFilesModule extends ReactContextBaseJavaModule
         }
 
         String sortOrder = MediaStore.Audio.Media.TITLE + " ASC";
-        Cursor musicCursor = musicResolver.query(musicUri, STAR, selection, null, sortOrder);
+        Cursor musicCursor = musicResolver.query(musicUri, null, selection, null, sortOrder);
 
         //Log.i("Tienes => ",Integer.toString(musicCursor.getCount()));
 
@@ -252,9 +255,11 @@ public class RNReactNativeGetMusicFilesModule extends ReactContextBaseJavaModule
                                             Bitmap songImage = BitmapFactory.decodeByteArray(albumImageData, 0, albumImageData.length);
 
                                             try {
-                                                String pathToImg = Environment.getExternalStorageDirectory() + "/" + songId + ".jpg";
-                                                encoded = fcm.saveImageToStorageAndGetPath(pathToImg, songImage);
-                                                items.putString("cover", "file://" + encoded);
+                                                // String pathToImg = Environment.getExternalStorageDirectory() + "/" + songId + ".jpg";
+                                                // encoded = fcm.saveImageToStorageAndGetPath(pathToImg, songImage);
+                                                // items.putString("cover", "file://" + encoded);
+                                                encoded = Base64.encodeToString(albumImageData, Base64.DEFAULT);
+                                                items.putString("cover", "data:image/jpeg;base64," + encoded);
                                             } catch (Exception e) {
                                                 // Just let images empty
                                                 Log.e("error in image", e.getMessage());
@@ -262,9 +267,14 @@ public class RNReactNativeGetMusicFilesModule extends ReactContextBaseJavaModule
 
                                             if (getBluredImages) {
                                                 try {
-                                                    String pathToImg = Environment.getExternalStorageDirectory() + "/" + songId + "-blur.jpg";
-                                                    blurred = fcm.saveBlurImageToStorageAndGetPath(pathToImg, songImage);
-                                                    items.putString("blur", "file://" + blurred);
+                                                    // String pathToImg = Environment.getExternalStorageDirectory() + "/" + songId + "-blur.jpg";
+                                                    // blurred = fcm.saveBlurImageToStorageAndGetPath(pathToImg, songImage);
+                                                    // items.putString("blur", "file://" + blurred);
+                                                    ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+                                                    songImage.compress(Bitmap.CompressFormat.JPEG, 60, byteArrayOutputStream);
+                                                    byte[] byteArray = byteArrayOutputStream.toByteArray();
+                                                    blurred = Base64.encodeToString(byteArray, Base64.DEFAULT);
+                                                    items.putString("blur", "data:image/jpeg;base64," + blurred);
                                                 } catch (Exception e) {
                                                     Log.e("error in image-blured", e.getMessage());
                                                 }
